RELEASE.md

```diff
@@ -102,12 +102,17 @@ git commit -m "Update dependencies"
 #### Updating React dependencies
 
 The React application recently moved to a monorepo system with multiple internal npm packages. Dependency upgrades are
-quite sensitive for the time being and should be done manually with caution. 
+quite sensitive for the time being.
 
-When you want to update a dependency, you have to go to every internal npm package where the dependency is used and
-manually change the version. Once you have taken care of that, you need to go back to `web/ui` and run `npm install`
+In case you want to update the ui dependencies, you can run the following command:
 
-**NOTE**: We are researching ways to automate and improve this.
+```bash
+make update-npm-deps
+```
+
+Once it is done, please verify that you still have one folder `node_modules` in the folder `web/ui`, and then run `make ui-build` to verify it's still working.
```

Comment:
To clarify, is the purpose here to check that there's only *one* folder vs. multiple (in the subdirectories)?

Reply:
the verification is more here to check if the UI is still building. It's not related to the fact you could have multiple `node_modules`.

The check of having multiple `node_modules` folder is more here to be sure the UI environment is still clean.
Usually when you have multiple  `node_modules` that means you have a dependency used with different version in the different module. Which can lead to some weird behaviours.

Does it answer your questions ?

Reply:
Hmmmm... then I think without further explanations, the first part of the sentence can be a bit confusing. How about:

```suggestion
Once this step completes, please verify that no additional `node_modules` directory was created in any of the module subdirectories (which could indicate conflicting dependency versions across modules). Then run `make ui-build` to verify that the build is still working.
```

Reply:
ah yeah far better.
Thanks @juliusv